### DISCUSSION TITLE
Auto import a copy of channels while building a mirth docker container/image

### DISCUSSION THREAD

**Discussion by kaustubhn** [nextgenhealthcare/connect#5077] [Q&A]
My goal is to build a docker container for mirth and import already exported mirth channels while building the container, I would like to ask the community if they can point me to some guide that will help me to achieve my goal.

Thanks in Advance.

**Comment by bartha-adam** [comment]
This is an interesting topic. Would also be interested.
@kaustubhn did you had any progress with the idea?

**Reply by kaustubhn** [reply]
@bartha-adam Not at present, we are importing channels manually, we tried scripts but they don't seem to work may be there is a specific way of importing using the command line cli.
